phpBB/language/en/acp/board.php

```diff
@@ -348,7 +348,7 @@
 
 // Cookie Settings
 $lang = array_merge($lang, array(
-	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'These details define the data used to send cookies to your users browsers. In most cases the default values for the cookie settings should be sufficient. If you do need to change any do so with care, incorrect settings can prevent users logging in. If you have problems with users staying logging in to your board, visit the <b><a href="https://www.phpbb.com/support/go/cookie-settings/">phpBB.com Knowledge Base - Fixing incorrect cookie settings</a></b>.',
+	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'These details define the data used to send cookies to your users browsers. In most cases the default values for the cookie settings should be sufficient. If you do need to change any do so with care, incorrect settings can prevent users logging in. If you have problems with users staying logging in to your board, visit the <b><a href="https://www.phpbb.com/support/docs/en/3.1/kb/article/fixing-incorrect-cookie-settings/">phpBB.com Knowledge Base > Fixing incorrect cookie settings</a></b>.',
```

Comment:
The link should be: https://www.phpbb.com/support/go/cookie-settings (without the trailing slash)

Reply:
Ah. Then let's just close the PR.

Reply:
Well I mean this pr should just remove that trailing slash I think.

Reply:
lol. Got it.